csharp/src/Ice/IceDiscovery/LocatorRegistry.cs

```diff
@@ -0,0 +1,283 @@
+// Copyright (c) ZeroC, Inc. All rights reserved.
+
+using System;
```

Comment:
Seems `using System` is no longer required in this file

Reply:
Fixed.